app/components/TimezoneDropdown.js

```diff
@@ -120,7 +120,7 @@ class TimezoneDropdown extends React.Component {
       let timez = this.props.targetTimezone;
       return (
         <div className={styles.timeDetail}>
-          {text} {sundial.formatInTimezone(new Date(), timez, 'h:mm a')}
+          {text} {sundial.formatInTimezone(new Date(), timez, 'LT')}
```

Comment:
Tämänhän voisi myös lähettää Tidepoolille, ihan pikkuisena erillisenä pull requestina. :)

Reply:
Ihan siihen alkuperäiseen toteutukseen liittyen.

Reply:
Joo. Tohon lokalisointiin liittyy joitakin muitakin paikkoja (ainakin lisää laitelistaa) mitkä pitäis viedä sinne jollain aikataululla...